Doc/tools/touch-clean-files.py

```diff
@@ -6,16 +6,27 @@
 
 from pathlib import Path
 
-# Input file has blank line between entries to reduce merge conflicts
-with Path("Doc/tools/clean-files.txt").open() as clean_files:
-    CLEAN = [
+# Exclude these whether they're dirty or clean,
+# because they trigger a rebuild of dirty files.
+EXCLUDES = {
+    Path("Doc/includes/wasm-notavail.rst"),
+    Path("Doc/whatsnew/changelog.rst"),
+}
+
+# Exclude "Doc/venv/"
+ALL_RST = {rst for rst in Path("Doc/").rglob("*.rst") if rst.parts[1] != "venv"}
```

Comment:
We have several other possible venv locations ignored in `.gitignore`:

https://github.com/python/cpython/blob/6d59c9e32ebb6d8468c7eb26e7cf3efd458c7a73/.gitignore#L55-L58

Should we exclude those too?

```suggestion
# Exclude "Doc/venv/" etc.
ALL_RST = {
    rst
    for rst in Path("Doc/").rglob("*.rst")
    if rst.parts[1] not in {"venv", ".venv", "env", ".env"}
}
```

Reply:
Sure, will do it in a slightly different way and also exclude the `include` subdir (suggested below).